### PR TITLE
Added Contributing.md file

### PR DESCRIPTION
**Pull request by muskankhedia**

### CODE REVIEW

File: website/content/docs/contribution.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+title: "Contribution"
+description: "Contributing Guidelines"
+
+weight: 5
+---
+
+First off, thanks for taking the time to contribute!
```
Comment (harkishen) [inline website/content/docs/contribution.md]: I think we shouldn't write this. They haven't contributed yet, rather are deciding to contribute. So something like `First off, thank you for showing your interest in contributing the project`

File: website/content/docs/contribution.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+The following is a set of guidelines for contributing to `Bench-routes`. These are mostly guidelines, not rules.
+Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+## Code of Conduct
+
+This project and everyone participates in it is governed by this [Code of Conduct](https://github.com/bench-routes/bench-routes/blob/master/CODE_OF_CONDUCT.md).
```
Comment (harkishen) [inline website/content/docs/contribution.md]:
```suggestion
This project and everyone who participates in it, are governed by this [Code of Conduct](https://github.com/bench-routes/bench-routes/blob/master/CODE_OF_CONDUCT.md).
```

File: website/content/docs/contribution.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+By participating, you are expected to uphold this code. Please report unacceptable behavior to [**benchroutes@gmail.com**](mailto:benchroutes@gmail.com).
```
Comment (harkishen) [inline website/content/docs/contribution.md]:
```suggestion
By participating, you are expected to uphold this code. Please report any unacceptable behaviour to [**benchroutes@gmail.com**](mailto:benchroutes@gmail.com).
```

File: website/content/docs/contribution.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+Unsure where to begin contributing? You can start by looking through these [`beginner`](https://github.com/bench-routes/bench-routes/issues?q=is%3Aissue+is%3Aopen+label%3Afeature) and [`good-first-issue`](https://github.com/bench-routes/bench-routes/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) issues:
+
+* Beginner issues - issues which should only require a few lines of code, and a test or two.
+
+#### Pull Request Process
+
+1. Make a new branch for each feature/pull request. 
+2. Ensure any install or build dependencies are removed before the end of the layer when doing a build.
+3. Ensure that the code follows the linting standards. `eslint`, `tslint` and `prettier` are used for the TypeScript files and `golint` is used for the Golang files. Use the command `make fix` to format the code.
```
Comment (harkishen) [inline website/content/docs/contribution.md:32]: I dont remember here, so will it be possible for you to please verify if we use all the three ways in the UI? The eslint`, `tslint` and `prettier`.

File: website/content/docs/contribution.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
+---
```
Comment (harkishen) [inline website/content/docs/contribution.md:1]: We can also include a point, in the end, saying if you want to contribute by discussing a possible feature, please consider opening a topic at our mailing list and then provide the mailing list link.

**Review by harkishen — CHANGES_REQUESTED** [review]
Some nits and a point. Rest looks perfect!

File: website/content/docs/contribution.md (hunk 4 above)
Comment (muskankhedia) [reply website/content/docs/contribution.md:32]: yes, we are using all three ways.

**Review by harkishen — APPROVED** [review]
LGTM. Please consider merging if this is ready.

**Pull request merged**